src/database/migrations/getMigrations.ts

```diff
@@ -0,0 +1,28 @@
+import { Payload } from '../..';
+import { MigrationData } from '../types';
+
+export async function getMigrations({
+  payload,
+}: {
+  payload: Payload;
+}): Promise<{ existingMigrations: MigrationData[], latestBatch: number }> {
+  const migrationQuery = await payload.find({
+    collection: 'payload-migrations',
+    sort: '-name',
+  });
+
+  const existingMigrations = migrationQuery.docs as unknown as MigrationData[];
+
+  // Get the highest batch number from existing migrations
+  const latestBatch = existingMigrations.reduce((acc, curr) => {
```

Comment:
Agreed, I can simplify this. Since we name the files based on ISO-8601 which can be lexicographically ordered, it should be safe. Since I'm sorting by `-name`, it'd be the first in the result in this case.